spa/src/components/Step/index.tsx

```diff
@@ -1,7 +1,9 @@
+
+/** @jsxImportSource @emotion/react */
```

Comment:
Can we add a ticket to apply this globally? Is this something we've looked into yet? CC @krazziekay